lib/msal-common/src/request/RequestValidator.ts

```diff
@@ -33,7 +33,8 @@ export class RequestValidator {
                 PromptValue.LOGIN,
                 PromptValue.SELECT_ACCOUNT,
                 PromptValue.CONSENT,
-                PromptValue.NONE
+                PromptValue.NONE,
+                PromptValue.CREATE
             ].indexOf(prompt) < 0
```

Comment:
Can we change this logic so we don't have to re-enumerate every PromptValue here?

Reply:
Sure. Done.